lab/03-testing.md

```diff
@@ -2,52 +2,73 @@
 
 # Testing
 
-We need to test that our django application works as expected. We have provided some tests that ensure that you can visit all of the urls and that the data is being returned as expected.
+We need to test that our Django application works as expected. We have provided some tests that ensure that you can visit all of the urls and that the data is being returned as expected.
 
-## Testing via Django Tests
-The first set of tests we'll run are the Django tests. These tests are run inside of the container and are a great way to ensure that your application is working as expected. Django does a fantastic job of setup and teardown of test resources 
+## Installing the prerequisites
 
+The Dev Container already has the production requirements for the Django app installed,
+but it doesn't have the development requirements yet: the linting and testing packages.
 
-navigate to the `demo-code` folder and run the following command:
+Install the development requirements via pip:
```

Comment:
Alternatively, this could go in devcontainer.json as a postAttach command

Reply:
+1 - Created a new issue for it.